.github/workflows/ci.yml

```diff
@@ -154,7 +154,7 @@ jobs:
         run: |
           conda list
           cd /tmp
-          pytest -vvv -r a --pyargs sunpy --cov-report=xml --cov=sunpy --cov-config=$GITHUB_WORKSPACE/pyproject.toml $GITHUB_WORKSPACE/docs -n auto
+          pytest -vvv -r a --pyargs sunpy --cov-report=xml --cov=sunpy --cov-config=$GITHUB_WORKSPACE/pyproject.toml $GITHUB_WORKSPACE/docs -n auto --color=yes
```

Comment:
I thought changes like this have to go into the template and be done by the bot?

Reply:
We don't have conda in the package template yet, mainly because I couldn't figure out how to generalise the dev env yaml.